### DISCUSSION TITLE
Adwaitapod 3.0 release delayed due to Rockbox

### DISCUSSION THREAD

**Discussion by D0-0K** [D0-0K/adwaitapod#19] [Announcements]
Unfortunately, while the theme is now complete, I've had to delay the release of the theme due to limitations imposed by the Rockbox themes uploader. Currently, both the file system layout of my assets, as well as the number of assets, are preventing the theme from being uploaded.

The first is unfortunate, but not a big loss to change. The second however is a dealbreaker. I'm 14 files over the limit, and trying to aim for their 100 file and folder limit would require massive changes and compromises which I just cannot see myself doing. 

Of course the FreshOS route is a possibility, uploaded a shell of a theme and telling users to come here and download, but I feel that's also too much work to do now and alienates some users who may wish to use the Utility to install. Instead, I've asked in the Rockbox IRC if a change can be made to raise the limit. I've located the config file that controls this, and made the change myself, however I feel this change is pretty significant and should be discussed with the devs before any major changes are submitted. This however is something I can't control, and as such I have no idea if, let alone when, this change will be implemented and so until then I simply cannot release the theme publically.

In the meantime I'd like to apologise to anyone who has been waiting for this to release.

**Comment by teacherguy2020** [comment]
Your themes deserve to be added to the Rockbox Utility. It’s not difficult to add them on our own, but as you mentioned, many people will only use the Utility

**Reply by D0-0K** [reply]
Thanks for reminding me of this post, it's in fact no longer an issue! I was able to submit a code change to update the theme uploader to support larger themes. The Rockbox team were very helpful when getting it merged :)
